.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -244,6 +244,8 @@ swap:
   pseudo ops|pseudoops: pseudo-ops
   pull-down: pulldown
   recognise: recognize
+  Red Hat OpenJDK: Red Hat build of OpenJDK
+  Red Hat Java|Red Hat java: Red Hat build of OpenJDK
```

Comment:
`ignorecase: true` - is set in this rule so I think Red Hat Java is all that's required. It will ignore case so upper and lower cases are found.

Reply:
@rohennes - Good point. No need to call it out so I removed it. Sorry bit rusty updating rules! Thanks for flagging.